lib/internal/child_process.js

```diff
@@ -510,7 +510,7 @@ function setupChannel(target, channel) {
     if (typeof callback === 'function') {
       process.nextTick(callback, ex);
     } else {
-      this.emit('error', ex);  // FIXME(bnoordhuis) Defer to next tick.
+      process.nextTick(emitErrorNT, this, ex);
```

Comment:
Sure, more explicit :+1: Saw this in a lot of other modules, so thought it was the preferred pattern.